### DISCUSSION TITLE
change stream

### DISCUSSION THREAD

**Discussion by InushaDeSilva** [Snowflake107/quickmongo#51] [Q&A] [answered]
is it possible to subscribe to a change stream and listen for an event when a value inside the database is updated?

**Comment by twlite** [comment, marked as answer]
Yes, this should be possible with
```js
const watcher = db.model.watch();
watcher.on("change", changes => console.log("Something changed", changes);
```

**Reply by InushaDeSilva** [reply]
thank you so much! it works

if you don't mind I have another question, is there a possibility of emitting custom events through the database and listening to it through a device connected to the same database. If it was possible I would imagine it would be similar to listening on data changed as done above.
